database/settings/2023_11_17_150202_readd_registration_links_to_general_settings.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Spatie\LaravelSettings\Migrations\SettingsMigration;
+
+return new class extends SettingsMigration
+{
+    public function up(): void
+    {
+        $this->migrator->add(
+            'general.individual_orientation',
+            [
+                'en' => 'https://share.hsforms.com/161eyaBsQS-iv1z0TZLwdQwdfpez',
+                'fr' => 'https://share.hsforms.com/161eyaBsQS-iv1z0TZLwdQwdfpez?lang=fr',
```

Comment:
@chosww if this French link was just for testing, we should remove it. I think you should be able to modify the value in the tests instead.